tooling/openshift_labeler/generate.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/sh
+
+file=$1
+hosts=/run/openshift/label.tmp.hosts
+inventory_file_path=$2
+label_prefix="role"
+pbench_label="type=pbench"
+inv_path=$3
+register_all_nodes=$4
+declare -a host
+declare -a group
+declare -a label
+
+# Check for kubeconfig
+if [[ ! -s $HOME/.kube/config ]]; then
+	echo "cannot find kube config in the home directory, please check"
+	exit 1
+fi
+
+# Check if oc client is installed
+which oc &>/dev/null
+echo "Checking if oc client is installed"
+if [[ $? != 0 ]]; then
+	echo "oc client is not installed"
+	echo "installing oc client"
+ 	curl -L https://github.com/openshift/origin/releases/download/v1.2.1/openshift-origin-client-tools-v1.2.1-5e723f6-linux-64bit.tar.gz | tar -zx && \
```

Comment:
Is it possible to store the oc client archive URL in a variable or command line argument, in case we want to install a specific version or if path changes ?

Reply:
Sure, added a variable which can be set in case we want to install a different version of oc client.